### ISSUE TITLE
Modify existing content security policy (CSP) headers

### ISSUE THREAD

**Issue by UNTRIVAL** [JingMatrix/ChromeXt#147]
When using Tampermonkey, some scripts require the option "Modify existing content security policy (CSP) headers" under the "Security" section in Tampermonkey settings to be set to "yes" in order to work properly on specific web pages. 
How can this be achieved with Chromext in Samsung Internet?

**Comment by JingMatrix** [comment]
No, this is currently impossible for ChromeXt.
For short future, I have no enough time to implement it yet.

**Comment by JingMatrix** [comment]
It seems that your goal is simply to remove the CSP restrictions.
I think that is not hard to implement.

**Comment by UNTRIVAL** [comment]
> It seems that your goal is simply to remove the CSP restrictions. I think that is not hard to implement.

Yes, you're correct. The focus is on removing the CSP restrictions.
